### PR TITLE
Settings: Update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by JohnsonRan**
It's been a long day

### CODE REVIEW

File: res/values-zh-rCN/yaap_strings.xml [hunk 1]
```diff
@@ -51,8 +51,8 @@
     <string name="update_settings_title">YAAP 更新</string>
     <string name="update_settings_summary">检查系统更新</string>
 
-    <!-- String for removal of sensitive info on about, depending on tap -->
-    <string name="device_info_protected_single_press">点击显示</string>
```
Comment (idoybh) [inline res/values-zh-rCN/yaap_strings.xml:55]: Did you remove these intentionally ?

File: res/values-zh-rCN/yaap_strings.xml [hunk 2]
```diff
@@ -226,8 +237,6 @@
     <!-- Connectivity check -->
     <string name="connectivity_check_title">网络连通性检查</string>
     <string name="connectivity_check_summary">选择用于网络连通性检查的 HTTP 站点</string>
-    <string name="connectivity_check_standard">默认 (Google)</string>
-    <string name="connectivity_check_disabled">禁用</string>
```
Comment (idoybh) [inline res/values-zh-rCN/yaap_strings.xml]: and these ?

File: res/values-zh-rCN/yaap_strings.xml (hunk 1 above)
Comment (JohnsonRan) [reply res/values-zh-rCN/yaap_strings.xml:55]: i moved it to [here](https://github.com/yaap/packages_apps_Settings/pull/45/files#diff-d37a05c519520f1f24d434c0fcf7ab773ec8147effea4b67e17528ea27d6742fR380)

File: res/values-zh-rCN/yaap_strings.xml (hunk 2 above)
Comment (JohnsonRan) [reply res/values-zh-rCN/yaap_strings.xml]: fixed, thanks!

File: res/values-zh-rCN/yaap_strings.xml (hunk 1 above)
Comment (idoybh) [reply res/values-zh-rCN/yaap_strings.xml:55]: OH! cool

File: res/values-zh-rCN/yaap_strings.xml (hunk 2 above)
Comment (idoybh) [reply res/values-zh-rCN/yaap_strings.xml]: Nice!

File: res/values-zh-rCN/yaap_strings.xml [hunk 3]
```diff
@@ -226,7 +237,9 @@
     <!-- Connectivity check -->
     <string name="connectivity_check_title">网络连通性检查</string>
     <string name="connectivity_check_summary">选择用于网络连通性检查的 HTTP 站点</string>
-    <string name="connectivity_check_standard">默认 (Google)</string>
+    <string name="connectivity_check_standard">Google</string>
+    <string name="connectivity_check_gos">GrapheneOS</string>
+    <string name="connectivity_check_miui">MIUI</string>
     <string name="connectivity_check_disabled">禁用</string>
```
Comment (idoybh) [inline res/values-zh-rCN/yaap_strings.xml]:
```suggestion
    <string name="connectivity_check_standard">默认 (Google)</string>
    <string name="connectivity_check_disabled">禁用</string>
```

there's no need to translate lines that are the same as original. and lets also keep the Chinese for "default" there. No?

**Review by idoybh — CHANGES_REQUESTED** [review]

**Comment by idoybh** [comment]
Awesomeness! Thank you! merging. Also btw you may wanna give me the translations for:

```xml
<!-- Minimum auto brightness -->
<string name="user_min_auto_brightness_title">Minimal auto brightness</string>
<string name="user_min_auto_brightness_summary">Caps auto brightness minimum value</string>
```

It's a feature I'm about to push. I will amend ur translations to my push

File: res/values-zh-rCN/yaap_strings.xml (hunk 3 above)
Comment (JohnsonRan) [reply res/values-zh-rCN/yaap_strings.xml]: sure! i forget to check original file😢

**Review by idoybh — APPROVED** [review]

**Comment by JohnsonRan** [comment]
```
<!-- Minimum auto brightness -->
<string name="user_min_auto_brightness_title">自动亮度最小值</string>
<string name="user_min_auto_brightness_summary">自动亮度调整时的亮度下限</string>
```
here it goes!

**Comment by idoybh** [comment]
> ```
> 
> <string name="user_min_auto_brightness_title">自动亮度最小值</string>
> <string name="user_min_auto_brightness_summary">自动亮度调整时的亮度下限</string>
> ```
> 
> here it goes!

thank you <3

**Pull request merged**